### ISSUE TITLE
Are DataTable Extensions usable with wb-tables?

### ISSUE THREAD

**Issue by hodge-jai** [wet-boew/wet-boew#8801]
I would like to add the Select extension(https://datatables.net/extensions/select/) to my table, which as of now is initialized via data-wb-tables. I've included the source files for the extension in my page, but adding "select":true to data-wb-tables doesn't allow rows to be selected. Is there a workaround for this? Am I doing something wrong?

**Comment by thekodester** [comment]
Unsure if this is the best method, but I got it working with this:

```
<head>
<!---OTHER CODE--->
<link rel="stylesheet" href="https://cdn.datatables.net/select/1.3.1/css/select.dataTables.min.css">
<!---OTHER CODE--->
</head>
<body>
<!---OTHER CODE--->
<table id="select" class="table table-striped table-hover">
<!---TABLE HTML-->
</table>
<!---OTHER CODE--->
<script>
wb.doc.on( "wb-ready.wb", function() {
  Modernizr.load( {
      load: [
        "site!deps/jquery.dataTables" + wb.getMode() + ".js",
        "https://cdn.datatables.net/select/1.3.1/js/dataTables.select" + wb.getMode() + ".js"
      ],
      complete: function() {
          let selector = "#select", table;
          table = $( selector );
          table.attr( "class", "table table-hover table-stripped wb-tables" );
          table.attr( "data-wb-tables", '{"select": true}' );
          table.trigger( "wb-init.wb-tables" );
      }
  } );
} );
</script>
</body>
```

Demo Picture:
<img width="1792" alt="Screen Shot 2020-01-09 at 12 05 51 AM" src="https://user-images.githubusercontent.com/23484460/72039624-cff83700-3273-11ea-818a-ba01b60b929b.png">

**Comment by duboisp** [comment]
Yes it is possible to use data table extension with the wet-boew table as @thekodester demo.

But @hodge-jai we didn't tested the data table with that extension activated. I recommend you to test if it work in all your supported browser and to perform a full WCAG 2.1 assessment of your pages that use that extension.

**Comment by duboisp** [comment]
@thekodester it will be interesting if you can add your above example in the data table documentation pages.

**Comment by hodge-jai** [comment]
@thekodester is there a way to apply the wet look and theme to a datatable that's already been initialized? Unfortunately my initialization is quite long, and uses quite a few single and double quotes, custom datatable buttons, as well as URLs that are resolved via Jinja templates. Passing all these parameters via a single string seems unwieldy and would require a lot of escaping.

**Comment by thekodester** [comment]
Not simply.

Best course of action if you wanna avoid triggers or rebuilding the table you need to write your own table function or css to match what WET does.

Also don’t forget you’ll need to do accessibility testing of the additional feature as @duboisp mentioned.

**Comment by GormFrank** [comment]
@hodge-jai Thank you @thekodester for the answer. @hodge-jai Feel free to close this is issue if the answers that were given works with you!
